Examples/Python/tests/test_examples.py

```diff
@@ -1132,7 +1132,8 @@ def test_full_chain_odd_example_pythia_geant4(tmp_path):
     # This test literally only ensures that the full chain example can run without erroring out
 
     # just to make sure it can build the odd
-    detector, trackingGeometry, decorators = getOpenDataDetector()
+    with getOpenDataDetector() as (detector, trackingGeometry, decorators):
```

Comment:
This is a leftover from https://github.com/acts-project/acts/pull/3471 - I saw the CI crashing again with a segfault because the ODD was not cleaned out correctly